pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -25,8 +28,17 @@ buildPythonPackage rec {
         --replace-fail '"setuptools>=64,<74"' '"setuptools"'
   '';
 
+  dependencies = [
+    typing-extensions
+  ];
+
   nativeBuildInputs = [
     setuptools
+    pythonRelaxDepsHook
+  ];
+
+  pythonRelaxDeps = [
+    "typing-extensions"
```

Comment:
Not needed in staging-next.

Reply:
I understand, but the version range specified by `pyzstd` is still a bit strict nonetheless. What if we want to backport this PR in the future for some reason?